### PR TITLE
Add image, links, text

### PR DESCRIPTION
**Pull request by ValdezBalderas**
All changes are cosmetic/textual/image content, none is functional/code:

- Added a figure (swing dancing, illustrating the ML task of video classification) to the README files.
- Tweaked the very first part of the README.
- Added mention of blog 1 and linked to it in README.
- Changed/added links to more friendly locations for the Kinetics dataset and I3D model. (They pointed to papers before, now they point to websites e.g. Deepmind). This was done on both README and notebook.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-We'll first describe what's here, then go through how to run everything.
+The primary contributions of this repository are
+
+1. A Jupyter Notebook [video_classification.ipynb](video_classification.ipynb)
+    - Demonstrating how to download, organize, explore and visuzlize the [Kinetics Human Action Video Dataset](https://deepmind.com/research/open-source/kinetics).
+    - Demonstrating how to download from the [Tensorflow hub](https://www.tensorflow.org/hub) a pretrained [I3D video classification model](https://deepmind.com/research/open-source/i3d-model), and test it on small samples of the Kinetics dataset. 
+    - Highlighting challenges and considerations of working with video data. 
+2. A script [scripts/evaluate.py](scripts/evaluate.py) that allows the user to perform evaluation of I3D on larger samples, or full splits, of the Kinetics dataset.
+
+> Check out our blog [An Introduction to Video Understanding: Capabilities and Applications](https://blog.fastforwardlabs.com/2021/12/14/an-introduction-to-video-understanding-capabilities-and-applications.html) for broad overview of the field to which video classification belongs.
```
Comment (melaniebeck) [inline README.md]: Small typo: "for broad overview" --> "for a broad overview"

File: README.md [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
-Additionally, this repository contains a script that allows the user to perform evaluation of a video classification model on some standardized video datasets.
+![Upsampling](images/video_classification_task.png)
 
-Instructions are given both for general use (on a laptop, say), and for Cloudera CML and CDSW.
-We'll first describe what's here, then go through how to run everything.
+The primary contributions of this repository are
+
+1. A Jupyter Notebook [video_classification.ipynb](video_classification.ipynb)
+    - Demonstrating how to download, organize, explore and visuzlize the [Kinetics Human Action Video Dataset](https://deepmind.com/research/open-source/kinetics).
```
Comment (melaniebeck) [inline README.md]: Small typo: "visuzlize" --> "visualize"

**Review by melaniebeck — CHANGES_REQUESTED** [review]
Looks good! Just a couple minor typos and it should be good to go.

File: README.md (hunk 1 above)
Comment (ValdezBalderas) [reply README.md]: Done, thank you!

File: README.md (hunk 2 above)
Comment (ValdezBalderas) [reply README.md]: Done, thank you!

**Pull request merged**